### PR TITLE
issue #75 removing required fields in the meta-schema

### PR DESCRIPTION
**Pull request by grahampacker-ms**
As discussed in #75 the required field is not behaving as expected. As it currently stands, the example patterns are invalid.

### CODE REVIEW

**Comment by grahampacker-ms** [comment]
@rocketstack-matt @jpgough-ms I think this needs a review from one of you

**Review by rocketstack-matt — APPROVED** [review]

**Comment by rocketstack-matt** [comment]
Not sure why Will's review wasn't enough. Will check.

**Comment by grahampacker-ms** [comment]
@rocketstack-matt Will isn't a codeowner on that folder, it requires a codeowner to approve

**Pull request merged**
